test/index.js

```
@@ -12,25 +12,20 @@ const Vision = require('vision');
 // Declare internals
 
 const internals = {
-    bootstrapServer(server, plugins, options, callback) {
+    async bootstrapServer(server, plugins, options) {
 
-        if (typeof options === 'function') {
-            callback = options;
-            options = {};
-        }
-
-        server.register([Inert, Vision].concat(plugins), options, (err) => {
-
-            if (err) {
-                return callback(err);
-            }
+        try {
```

Useless try/catch, just let the error bubble.